### PR TITLE
Jetpack: 11.2-a.3 changelog editorial

### PR DESCRIPTION
**Pull request by coder-karen**
#### Changes proposed in this Pull Request:

* Jetpack: 11.2-a.3 changelog and readme editorial

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proof-read the changelog and readme.

### CODE REVIEW

File: projects/plugins/jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -4,35 +4,27 @@
...
-- VideoPress: Implement upload progress bar [#24953]
-- VideoPress: Improve upload error layout [#24963]
-
-### Improved compatibility
-- add videopress package as dependency [#25045]
+- Podcast Player: add new actions to make it possible for users to set up code that runs for podcast fetches. [#25046]
+- VideoPress: add upload progress bar. [#24953]
+- VideoPress: improve the UX when rendering the block. [#24891]
+- VideoPress: improve upload error layout. [#24963]
+- VideoPress: limit preview attempts. [#24897]
```
Comment (samiff) [inline projects/plugins/jetpack/CHANGELOG.md]: If these are all for the new VP block, perhaps these could be merged and mention the block is in beta?

Comment (coder-karen) [reply projects/plugins/jetpack/CHANGELOG.md]: Oh they are - the functionality doesn't exist with regular testing on WoA actually so I wonder if they should be removed entirely too...

Comment (samiff) [reply projects/plugins/jetpack/CHANGELOG.md]: I think merging to one line would be good since with the beta plugin users can use the Atomic releases.

Comment (coder-karen) [reply projects/plugins/jetpack/CHANGELOG.md]: Agree, added a new commit: 85f905f11dc6939c3a365b0385c3281b400344d9

**Review by samiff — APPROVED** [review]

**Comment by coder-karen** [comment]
r248894-wpcom

**Pull request merged**